src/Machine.js

```diff
@@ -222,6 +222,9 @@ class Machine {
       this._includeMacro(macro, context, buffer);
     } else {
       // source inclusion
+
+      // replace for compatibility with Windows
+      instruction.value = instruction.value.replace(/\\/g, "/");
```

Comment:
I believe this is not the right place to do the conversion. It should be done on after expression is evaluated.